### DISCUSSION TITLE
32

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#174] [General]
# 潮流周刊第32期 - 南京

封面图摄于南京总统府，有个城市旅游套路就是前几天吃吃逛逛，最后一天叫个跟团游，带你玩遍整个城市知名景点，车接送还很便宜，最后送你回高铁站，很香；此外南京是一座我觉得很潮流的城市，很值得一去，推荐去吃吃馨芳园、澳门轩、递酒烤场、哎抖猪蹄、郭阿姨月亮馍，然后逛逛新街口。 

https://weekly.tw93.fun/posts/32-%E5%8D%97%E4%BA%AC

**Comment by dagmom** [comment]
没去 1701 吗
